pkcs7.go

```diff
@@ -85,7 +85,7 @@ type recipientInfo struct {
 type encryptedContentInfo struct {
 	ContentType                asn1.ObjectIdentifier
 	ContentEncryptionAlgorithm pkix.AlgorithmIdentifier
-	EncryptedContent           asn1.RawValue `asn1:"tag:0,optional,explicit"`
+	EncryptedContent           asn1.RawValue `asn1:"tag:0,optional"`
```

Comment:
The merges look good to me, I see it's all from upstream. I do have one-and-a-half suggestions, basically the only difference with upstream code is around line 171, the upstream version is below.

Georg made the changes as it looked the original upstream did not work on large files, hence the looping until everything is parsed and appending with `content = append(content, parsedContent...)`.
I suggest you make it explicit, maybe with comments in the code, that this is where we differ from upstream. And maybe, just maybe, the normal upstream version now works on bigger files (although I didn't check if relevant changes were made to asn1.Unmarshal)?

```	
// Compound octet string
	if compound.IsCompound {
		if _, err = asn1.Unmarshal(compound.Bytes, &content); err != nil {
			return nil, err
		}
	} else {
```

Reply:
Ok, i will add a comment to mark the change we did compared to upstream.
I have also testing with the upstream code without the changes Georg made but than some unit tests are failing. So this change is required.